plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -362,8 +363,8 @@ class FeatureDevController(
 
         newTabOpened(tabId)
 
-        messenger.sendAnswer(tabId = tabId, messageType = FeatureDevMessageType.Answer, message = message("amazonqFeatureDev.chat_message.ask_for_new_task"))
```

Comment:
I think we need this message to where we handle NEW_TASK followup.

https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt#L115

Reply:
Looks there are still some build issues. I think we need update FeatureDevControllerTest

Reply:
Sorry, I'm confused. You've said that we should remove this one. So what you're saying is that we should remove only for the exception?

Reply:
> Looks there are still some build issues. I think we need update FeatureDevControllerTest

Yes, because we've removed the "ask_for_new_task"

Reply:
Added a "validation"